core/src/event/rindexer_event_filter.rs

```diff
@@ -180,25 +180,25 @@ impl RindexerEventFilter {
 
     pub fn topic1(&self) -> Topic {
         match self {
-            RindexerEventFilter::Address(filter) => filter.topics[1].clone(),
-            RindexerEventFilter::Filter(filter) => filter.topics[1].clone(),
-            RindexerEventFilter::Factory(filter) => filter.topics[1].clone(),
+            RindexerEventFilter::Address(filter) => filter.topics[0].clone(),
```

Comment:
was there a reason we changed this? was it breaking @zannis ?

Reply:
discussed and reverted